### ISSUE TITLE
HTMX script does not load in block editor causing unwanted behavior

### ISSUE THREAD

**Issue by jasalt** [svandragt/htmxpress-serverside-block#2]
The HTMX template shows up in block editor quite nicely but if it has form with submit button, clicking the submit will exit block editor. Also `hx-indicator` attribute is failing. In example video I have following simple form where `/htmx/api_results` returns some simple html into the target div: 

```
<div style='border:2px groove black; padding:5px;'>
	<form hx-post="/htmx/api_results"
		hx-target="#api_results_div"
		hx-swap="scroll:bottom"
		hx-indicator="#loading-indicator">
		<input name="htmxPostID" placeholder="give url" type=text>
		<button>Fetch</button>
	</form>
	<h3>Results</h3>
	<div id="api_results_div"></div>

	<img id="loading-indicator" alt="Result loading..." class="htmx-indicator" width="50" src="<?php echo esc_url($image_url) ?>"/>
</div>
```
https://github.com/svandragt/htmxpress-serverside-block/assets/2306521/4c8caf51-0370-4746-a971-cc5d8ad0075b

Would there be a reason to force loading HTMX script in block editor, or would it be better to render buttons or whole htmx-block content there as disabled?

Thought to note this up while repo is experimental still, I did not expect it all work. Will probably continue experimenting with shortcode to keep it more simple for now.

Tested with [this fork](https://github.com/jasalt/htmxpress-serverside-block/commit/e20a8f8058dde1d5ea76829f9acd904da00de1dc) having a couple other modifications for block registration issue I had, but guessing main repo behaves same way.

**Comment by svandragt** [comment]
HTMXpress is not enqueued by default in the block editor, that's why you're seeing issues. I've made a note to review it on the htmxpress repo. Thanks for the report and apologies for taking so long to get round to it.

**Comment by jasalt** [comment]
@svandragt Thank's, no problem. I've moved to do custom setup in projects using Htmx (or Datastar more lately) but your HTMXPress repos have been an inspiration.

**Comment by svandragt** [comment]
Yeah I've not had much chance to use HTMX at work only in personal projects and they tend not to be WP these days. Thanks for your kind words!
